src/foam/core/Action.js

```diff
@@ -153,9 +153,7 @@ foam.CLASS({
     },
 
     function isEnabledFor(data) {
-      return this.isEnabled ?
-        data.slot(this.isEnabled).get() :
-        true ;
+      return foam.Function.withArgs(this.isEnabled, data);
```

Comment:
I prefer to have isEnabled be null when not used rather than an empty function because it avoids the not negligible amount of work to hookup and then remove the listeners when not needed.

Reply:
Ok, I'll revert the change I made a week or two ago that sets the default functions and update everywhere they're referenced to do null checks.

Reply:
Resolved by 96c3728